spec/requests/api/v1.0/service_plans_spec.rb

```diff
@@ -55,6 +55,10 @@
       it "returns the newly created ServicePlan in an array" do
         expect(json.first["create_json_schema"]).to eq topo_service_plan.create_json_schema
       end
+
+      it "shows modified as false" do
+        expect(json.first["modified"]).to be_falsey
```

Comment:
Should we have a spec that shows when `@modified` is truthy?

Reply:
Yep definitely - added.